4_model_evaluation.yml

```diff
@@ -0,0 +1,55 @@
+target_default: 4_model_evaluation
+
+packages:
+  - dplyr
+  - scipiper
+
+sources:
```

Comment:
Are you using all of these sources in this yml? Seems like you only need two of these?